microsoftgraph/msgraph-cli-archived
# Feat/nest commands

## Overview

Fixes #64 
Fixes #65 

### Demo

N/A

### Notes

I disabled `long_summary` section of help information because it prevented the customer from seeing all available parameters, this was a necessary tradeoff. 

## Testing Instructions

* Pull these changes
* Run `./mg users user-user create-user -h` to see the list of parameters
